### PR TITLE
fix: forcing the light theme for the app

### PR DESCRIPTION
**Pull request by acezard**
In order to avoid issues with unhandled dark modes,
we disable the DayNight compat of the app
and replace it with a Light compat only.
This will disable algorithmic darkening

Tested with forceDarkMode on WebView on a Android phone in dark theme with a dark themed chrome: the homepage is not dark despite not having a meta color-scheme

### CODE REVIEW

File: android/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <resources>
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
+    <!-- Forcing the light theme for the entire app -->
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```
Comment (Crash--) [inline android/app/src/main/res/values/styles.xml:4]: Do you know the difference between that and        ` <item name="android:forceDarkAllowed">false</item>` ?https://github.com/facebook/react-native/issues/27876

Comment (acezard) [reply android/app/src/main/res/values/styles.xml:4]: @Crash-- from what I understand we should use both to be sure. They seem to do the same thing but in a different way (to me). It's possible some devices may work or not in one mode or another.

Comment (acezard) [reply android/app/src/main/res/values/styles.xml:4]: updated

**Review by Crash-- — APPROVED** [review]

**Comment by acezard** [comment]
> @acezard Let's merge it. Once it's merged, can you send me a built APK please? Like the one we send to the play store please

ok

**Pull request merged**